doc/manpages.py

```diff
@@ -18,6 +18,8 @@
 # - Manual section
 man_pages = [
     ('man1/flux-broker', 'flux-broker', 'Flux message broker daemon', [author], 1),
```

Comment:
Could we maybe add a note here that the order affects `flux help` output?

Someone might decide to alphabetize the list or something.

Reply:
Good suggestion. I went ahead and did that.